pkg/workspace.go

```diff
@@ -141,17 +162,29 @@ func (c *Client) GetWorkspace(namespace, uid string) (workspace *Workspace, err
 	if err != nil {
 		return
 	}
+
 	workspace = &Workspace{}
 	if err = c.DB.Get(workspace, query, args...); err == sql.ErrNoRows {
 		err = nil
 		workspace = nil
+
+		return
 	}
-	if workspace != nil {
-		if err = json.Unmarshal(workspace.ParametersBytes, &workspace.Parameters); err != nil {
-			return
-		}
+	if err != nil {
```

Comment:
This change won’t work because both `err` and `workspace` can be `nil` when a workspace is not found

Reply:
Ah, I missed the return above.